pkg/occlient/occlient.go

```diff
@@ -444,7 +444,11 @@ func (c *Client) GetImageStreamsNames(namespace string) ([]string, error) {
 
 // NewAppS2I create new application using S2I
 // if gitUrl is ""  than it creates binary build otherwise uses gitUrl as buildSource
-func (c *Client) NewAppS2I(name string, builderImage string, gitUrl string, labels map[string]string, annotations map[string]string) (string, error) {
+func (c *Client) NewAppS2I(applicationName, componentName string, builderImage string, gitUrl string, labels map[string]string, annotations map[string]string) (string, error) {
```

Comment:
anything `occlient` shouldn't have any references to component or application. We should have clean abstraction hierarchy.

Reply:
No worries, I'll change this up to be higher up in the code :+1: you're right. It shouldn't have any references to neither of those (especially since we're just combining the strings)